com.epicgames.Fortnite/com.epicgames.Fortnite.yml

```diff
@@ -99,7 +142,7 @@ modules:
           - wine64 reg add 'HKEY_CURRENT_USER\Software\Wine\DllOverrides' /v shcore /d "" /f
           - 
           - echo "Installing application..."
-          - msiexec /i "${XDG_CACHE_HOME}/fortnite-installer.msi"
+          - wine64 msiexec /i "${XDG_CACHE_HOME}/fortnite-installer.msi"
```

Comment:
Will probably need `-SkipBuildPatchPrereq` & `-opengl`? Most of the talk I see online seems to suggest you need it.